wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -32,6 +33,8 @@ var (
 	HTTPHeaderFeedClientVersion       = textproto.CanonicalMIMEHeaderKey("Arbitrum-Feed-Client-Version")
 	HTTPHeaderRequestedSequenceNumber = textproto.CanonicalMIMEHeaderKey("Arbitrum-Requested-Sequence-Number")
 	HTTPHeaderChainId                 = textproto.CanonicalMIMEHeaderKey("Arbitrum-Chain-Id")
+	upgradeToWSTimer                  = metrics.NewRegisteredTimer("arb/wsbroadcastserver/wsupgrade/duration", nil)
+	startWithHeaderTimer              = metrics.NewRegisteredTimer("arb/wsbroadcastserver/startwithheader/duration", nil)
```

Comment:
To better fit with existing metrics, how about something like
`"arb/feed/clients/wsupgrade/duration"`?

Reply:
makes sense!